api/src/main/java/net/starype/quiz/api/game/answer/MultipleChoiceValidity.java

```diff
@@ -0,0 +1,23 @@
+package net.starype.quiz.api.game.answer;
+
+public class MultipleChoiceValidity implements ValidityEvaluator {
+
+    private static MultipleChoiceValidity instance = null;
+
+    private MultipleChoiceValidity() {
+
+    }
+
+    public static synchronized MultipleChoiceValidity getInstance() {
```

Comment:
```suggestion
    public static MultipleChoiceValidity getInstance() {
```
so you have chosen perpetual war I see